src/sets.jl

```diff
@@ -295,8 +295,8 @@ end
 function projection_on_set(::DefaultDistance, ::MOI.PositiveSemidefiniteConeTriangle, z::Array{Float64})
     dim = Int64(floor(√(2*length(z))))
     X = unvec_symm(z, dim)
-    λ, U = eigen(X)
-    return vec_symm(U * Diagonal(max.(λ,0)) * U')
+    λ, U = LinearAlgebra.eigen(X)
+    return vec_symm(U * LinearAlgebra.Diagonal(max.(λ,0)) * U')
```

Comment:
This could be on two lines for clarity
```suggestion
    D = LinearAlgebra.Diagonal(max.(λ, 0))
    return vec_symm(U * D * U')
```

Reply:
ok